tests/unittests/test_fuelTanks.py

```diff
@@ -0,0 +1,268 @@
+import unittest

```

Comment:
Very cool that you added this! I assume this isn't used anywhere but you wrote it for local testing? It would be supercool to have Python tests in CI...

Reply:
I made it similar to the existing python test, which is still based on the old TiGL version, though:

https://github.com/DLR-SC/tigl/blob/ab8a1a443fd50020578f3a42e7e6d887e89cdc06/tests/unittests/pythonwrapper-tests.py#L21

Yep, then I used a local Python interpreter to run the tests. Would be nice to have it also in CI (see #1012 and #826).